astropy/stats/sigma_clipping.py

```diff
@@ -465,7 +465,7 @@ def __call__(self, data, axis=None, masked=True, return_bounds=False,
 
         Returns
         -------
-        result : flexible
+        result : array_like
```

Comment:
`array-like`

Reply:
technically both work, but for consistency and personal preference, I agree that ``array-like`` is better.